tests/rptest/services/templates/omb_driver.yaml

```diff
@@ -2,9 +2,14 @@ name: Redpanda-{{name}}
 driverClass: io.openmessaging.benchmark.driver.redpanda.RedpandaBenchmarkDriver
 
 replicationFactor: {{replication_factor}}
-reset: true
+reset: {{reset or 'true'}}
```

Comment:
Especially after this change I question the value of this jinja template. We mostly just take dicts and slap them into the different sections. It seems like this could be entirely in python at considerably less complexity.

Reply:
I agree, though at this point it'd take longer to implement that then to just go with this.